packages/editor-ui/src/components/WorkflowActivator.vue

```diff
@@ -74,16 +80,28 @@ export default mixins(
 					}
 					return '#13ce66';
 				},
+				disabled(): boolean {
+					if (this.isCurrentWorkflow) {
+						// Switch should be enabled if workflow is currently active
+						return !this.workflowActive && !this.containsTrigger;
+					}
+					return false;
```

Comment:
```suggestion
				return this.isCurrentWorkflow && !this.workflowActive && !this.containsTrigger;
```

Reply:
I think `if` improves readability, but yours is more succinct!